modular-particle-system/src/generators/shapeGenerator.ts

```diff
@@ -34,11 +40,16 @@ import { deserializeShape, getRandomPositionInsideShape, serializeShape, Shape }
  */
 export class ShapeGenerator extends ParticleGenerator {
     shape?: Shape;
+    edgesOnly?: boolean;
```

Comment:
Should be just boolean with default value = false

Reply:
yeps